psi4/src/export_fock.cc

```diff
@@ -102,6 +102,11 @@ void export_fock(py::module &m) {
         .def("denominator_occ", &LaplaceDenominator::denominator_occ, "docstring")
         .def("denominator_vir", &LaplaceDenominator::denominator_vir, "docstring");
 
+    py::class_<TLaplaceDenominator, std::shared_ptr<TLaplaceDenominator>>(m, "TLaplaceDenominator", "docstring")
+        .def(py::init<std::shared_ptr<Vector>, std::shared_ptr<Vector>, double>())
+        .def("denominator_occ", &TLaplaceDenominator::denominator_occ, "docstring")
+        .def("denominator_vir", &TLaplaceDenominator::denominator_vir, "docstring");
```

Comment:
Fill in the docstring placeholders, please

Reply:
There is nothing filled in for `LaplaceDenominator` either, should I fill it in for those also?

Reply:
Sure, filling in any placeholders you have a good notion of always appreciated.

Reply:
Comments added